### DISCUSSION TITLE
Justifying FormattedText

### DISCUSSION THREAD

**Discussion by DanielTyebkhan** [AvaloniaUI/Avalonia#8826] [Q&A] [answered]
I am trying to draw some justified text in a custom Control, but it doesn't seem to be working as I would expect. The text wraps, but changing the FormattedText.TextAlignment property doesn't change the output. I am currently on the v11 preview version. Any help would be appreciated. Thanks. 

Code is as follows:

    public class TestPanel : Panel
    {
        public const string TestText = "some text some text some text some text some text some text some text some text some text some text some text some text some text some text some text some text";
        public const int TestWidth = 200;
        public TestPanel()
        {
            Children.Add(new TestControl());
        }
    }
        
    public class TestControl: Control {
        public override void Render(DrawingContext context)
        {
            var ft = new FormattedText(TestPanel.TestText, CultureInfo.CurrentCulture, FlowDirection.LeftToRight, Typeface.Default, 12, new SolidColorBrush(Colors.Black))
            {
                TextAlignment = TextAlignment.Justify,
                MaxTextWidth = TestPanel.TestWidth,
            };
            context.DrawText(ft, new(0,0));
        }
    }

**Comment by Gillibald** [comment, marked as answer]
Only TextLayout supports TextAlignment.Justify

**Comment by simonegli8** [comment]
Are there any plans for this to be implemented in the future?
